vavr/src/main/java/io/vavr/control/Validation.java

```diff
@@ -393,36 +402,36 @@ default boolean isEmpty() {
     T get();
 
     /**
-     * Gets the value if it is a Valid or an value calculated from the error
+     * Gets the value if it is a Valid or an value calculated from the errors
      *
-     * @param other a function which converts an error to an alternative value
+     * @param other a function which converts an errors to an alternative value
      * @return the value, if the underlying Validation is a Valid, or else the alternative value
-     * provided by {@code other} by applying the error.
+     * provided by {@code other} by applying the errors.
      */
-    default T getOrElseGet(Function<? super E, ? extends T> other) {
+    default T getOrElseGet(Function<Seq<? extends E>, ? extends T> other) {
         Objects.requireNonNull(other, "other is null");
         if (isValid()) {
             return get();
         } else {
-            return other.apply(getError());
+            return other.apply(getErrors());
         }
     }
 
     /**
-     * Gets the error of this Validation if is an Invalid or throws if this is a Valid
+     * Gets the errors of this Validation if is an Invalid or throws if this is a Valid
      *
-     * @return The error of this Invalid
+     * @return The errors of this Invalid
      * @throws RuntimeException if this is a Valid
      */
-    E getError();
+    Seq<? extends E> getErrors();
```

Comment:
The lack of declaration-site variance makes type declarations hard in Java. Here, the return type has to be

```java
Seq<E> getErrors();
```

because we do not plan to override the Validation class (and return a subtype of Seq in a method override).